utils/unittest/unittest.go

```diff
@@ -394,19 +394,19 @@ func NetworkTopology() network.Topology {
 
 // CrashTest safely tests functions that crash (as the expected behavior) by checking that running the function creates an error and
 // an expected error message.
-func CrashTest(t *testing.T, scenario func(*testing.T), expectedErrorMsg string, testName string) {
+func CrashTest(t *testing.T, scenario func(*testing.T), expectedErrorMsg string) {
```

Comment:
since we're no longer checking for exit 1, we should pass in the expected set of exit status as well?

i.e., `func CrashTest(t *testing.T, scenario func(*testing.T), expectedErrorMsg string, expectedStatus int, additionalExpectedStatus ...int)`

Reply:
alternatively, 

```
func CrashTest(
    t *testing.T, scenario func(*testing.T),
    expectedErrorMsg string,
    expectedStatus ...int)
// exit code must be in expectedStatus if it's non-empty.  otherwise, it must be non-zero
```

Reply:
I just your comment @pattyshack and I already commented on this in the PR, but why would we pass in a set of expected exit statuses, instead of just a single expected status? Also, why would we make it possible to expect ANY non-zero exit code? That seems too permissive to me because we could miss an error that we weren't expecting.

Reply:
Good point.  Let me remove this.